srcpkgs/base-chroot/template

```diff
@@ -17,6 +17,6 @@ esac
 depends+="
  base-files binutils gcc gcc-ada libada-devel
  patch sed findutils diffutils make gzip coreutils
- file bsdtar ccache xbps mpfr ncurses libreadline8
+ file bsdtar ccache rust-sccache>=0.2.15 xbps mpfr ncurses libreadline8
```

Comment:
nope

Reply:
Why?

Reply:
base-chroot only depends on bootstrap packages and sccache is not one, and will not be one, as that'd impose rust toolchain dependency for bootstrap and we!re not doing that

besides, sccache is limited to a small fraction of CPU architectures, which is another reason even if rust was bootstrap

Reply:
OK, I'll remove `rust-sccache` from the dependencies. Instead, I'll add a note into README.md that in order to use `sccache` with `xbps-src`, one should run `xbps-install -r masterdir/ rust-sccache`.

Reply:
> besides, sccache is limited to a small fraction of CPU architectures, which is another reason even if rust was bootstrap

AFAIK plain `sccache` (local cache) should work as long as you have Rust for the architecture. `sccache-dist` (distributed compilation) works on Linux x86_64 only, but this PR isn't about `sccache-dist`.

Reply:
it doesn't, read the template you are updating

Reply:
(but also we're not gonna make rust toolchain a bootstrap package, so the point is moot)

Reply:
also, installing manually into masterdir won't work either

if you want to do this you will need to find a way to have the masterdir lazily install it the first time you use it, similarly to the crosstoolchain and so on

Reply:
Have the cargo build style pull it in if we aren't building `rust`, `cargo` or `sccache`? With a `rust_bootstrap` flag or whatever.

Reply:
> also, installing manually into masterdir won't work either
> 
> if you want to do this you will need to find a way to have the masterdir lazily install it the first time you use it, similarly to the crosstoolchain and so on

I've just checked it out: `./xbps-src clean` doesn't remove manually installed `sccache` from the `masterdir/`. `hostdir/sccache` is also preserved. Then if I try to build some Rust package (e. g. `fd`), `SCCACHE_DIR=hostdir/binpkgs sccache -s` reports cache hits.

https://user-images.githubusercontent.com/29541480/117307385-3356a100-ae89-11eb-9f77-44f8c6b50de1.mp4